aten/src/ATen/native/native_functions.yaml

```diff
@@ -5018,6 +5018,24 @@
   device_check: NoCheck
   device_guard: False
 
+- func: sym_size.int(Tensor self, int dim) -> SymInt
```

Comment:
Is the main reason for adding these to native_functions so that we can add (canonical)? Or is there anything else.


Nit: this will also expose these under the torch namespace, torch.sym_size. Do we want that?

Reply:
My intention for adding them is to mark them as canonical, and publish a schema for these ops. 

I actually don't know if it matters for torchgen to work for these ops, as they already have special handling in python_variable.py. 

Yes, I don't need to expose them as torch.sym_size, I just need them available as torch.ops.aten.sym_size.

Reply:
But I am not able to make the CI happy... 
 
I run into compiling problems for generated header files: https://gist.github.com/SherlockNoMad/19e9d618c22f57cc2d7eaf6700079fd0